doc/rst/language/spec/ranges.rst

```diff
@@ -654,7 +659,9 @@ following primary properties:
    ``R by -1`` to contain the ordered sequence :math:`3,2,1`. But then
    ``R by -1`` would be different from ``3..1 by -1`` even though it
    should be identical by substituting the value in R into the
-   expression.
+   expression.  We have also found that using a strict ordering of low
+   and high bounds can be clearer when using ranges to slice
+   multidimensional arrays.
```

Comment:
What is so special about multidimensional arrays here?

Reply:
I find that with just 1D it's easier to keep track of up vs. down whereas with a multidimensional array where each dimension might be counting up or down, it makes my head spin more.  But I'm fine removing multidimensional as well.